README.md

```diff
@@ -17,17 +17,17 @@ This utility measures the accuracy of detecting malicious versus benign prompts.
 - Poetry v1.x or greater
 - Pangea's Prompt Guard:
    1. Sign up for a free [Pangea account](https://pangea.cloud/signup).
-   1. After creating your account and first project, skip the wizards. This will take you to the Pangea User Console, where you can enable the service.
-   1. Click Prompt Guard in the left-hand sidebar.
-   1. In the service enablement dialogs, click **Next**, then **Done**.
-   1. Click **Finish** to go to the service page in the Pangea User Console.
-   1. On the **Overview** page, capture the following **Configuration Details** by clicking on the corresponding values:
-      - **Domain** - Identifies the cloud provider and is shared across all services in a Pangea project.
+   2. After creating your account and first project, skip the wizards. This will take you to the Pangea User Console, where you can enable the service.
+   3. Click Prompt Guard in the left-hand sidebar.
+   4. In the service enablement dialogs, click **Next**, then **Done**.
+   5. Click **Finish** to go to the service page in the Pangea User Console.
+   6. On the **Overview** page, capture the following **Configuration Details** by clicking on the corresponding values:
+      - **Base URL** - The full base URL for Prompt Guard (e.g. "https://prompt-guard.aws.us.pangea.cloud"). This must be set using the `PANGEA_BASE_URL` environment variable.
```

Comment:
Sorry for not being clear - we can't copy base URL from PUC, which is what we describe here. So, it has to be **Domain**, but then we use it in the base URL. 

I can make that change is this is not clear. I need to make some other updates anyway, and I already made them locally.